### DISCUSSION TITLE
Docs convert docker to containerd?

### DISCUSSION THREAD

**Discussion by Fyne5** [k3s-io/k3s#5609] [Q&A]
Dear friends,

I'm using k3s_1.19 with docker driver, I'm planning upgrade k3s to 1.20 and with containerd driver, but I searched and not found docs at: https://rancher.com/docs/k3s/latest/en/installation/installation-requirements/

Could you please let me know how can I upgrade and convert from docker to containerd?

Thanks so much.

**Comment by brandond** [comment]
* remove --docker from the k3s arguments
* stop k3s
* stop docker
* start k3s

**Reply by Fyne5** [reply]
Thanks brandond,

After I re-run this command which removed --docker and when k3s, it still keep docker
```
curl -sfL https://get.k3s.io | INSTALL_K3S_EXEC="server --container-runtime-endpoint unix://--flannel-iface ens192 --advertise-address 10.10.8.150 --node-ip 10.10.8.150" INSTALL_K3S_VERSION="v1.24.11+k3s1" sh -
```

So, what is exactly of "remove --docker"? Thanks so much
